### PR TITLE
print headers on page break

### PR DESCRIPTION
**Pull request by shoaib-moeen**

### CODE REVIEW

**Comment by Lucas-C** [comment]
Hi @shoaib-moeen

Note that the table headers already split over page breaks:
* https://github.com/py-pdf/fpdf2/blob/master/test/table/test_table.py#L333
* https://github.com/py-pdf/fpdf2/blob/master/test/table/table_with_single_top_line_layout_and_page_break.pdf

Could you please provide a minimal code snippet reproducing the problem that you are facing? 🙂

**Comment by shoaib-moeen** [comment]
Hi @Lucas-C ,

Problem was when a single row breaks the page due to very long text it didn't print headers on the next page. It is related to issue #1142  . For now my usage was really limited to the headers not being displayed. I ll try to resolve that issue as well.

**Comment by Lucas-C** [comment]
> Problem was when a single row breaks the page due to very long text it didn't print headers on the next page. It is related to issue #1142 . For now my usage was really limited to the headers not being displayed.

Alright.
Could you maybe provide some minimal code reproducing your problem? 🙂
_cf._ https://stackoverflow.com/help/minimal-reproducible-example

File: fpdf/table.py [hunk 1]
```diff
@@ -501,6 +501,7 @@ def _render_table_cell(
                     wrapmode=self._wrapmode,
                     padding=padding,
                     link=cell.link,
+                    page_break_buffer=10,
```
Comment (gmischler) [inline fpdf/table.py:504]: There should be no "magical numbers" like that in the code.
How did you arrive at exactly this size?
Are you aware that size specifications are relative to the document units? If someone uses inches as document units, this turns into half a page!

**Comment by Lucas-C** [comment]
Hi!

Just a quick update about this: @shoaib-moeen are you still willing to work on this feature?
If you don't, or do not have time for this anymore, that totally fine! 🙂
If so we will just close this PR.

Also, if you need feedback or help on this PR, please tell us! 🙂

**Comment by Lucas-C** [comment]
Closing this PR for now.
Please comment there if you want to reopen it.

**Pull request closed without merging**